test/m4/appframework/appframework_test.go

```diff
@@ -366,174 +379,187 @@ var _ = Describe("m4appfw test", func() {
 			}
 
 			// Deploy M4 CRD
-			testenvInstance.Log.Info("Deploy Multisite Indexer Cluster with SHC")
+			testenvInstance.Log.Info("Deploy Multisite Indexer Cluster with Search Head Cluster")
 			siteCount := 3
 			shReplicas := 3
 			indexersPerSite := 1
 			err = deployment.DeployMultisiteClusterWithSearchHeadAndAppFramework(deployment.GetName(), indexersPerSite, siteCount, appFrameworkSpec, true, mcName, "")
-			Expect(err).To(Succeed(), "Unable to deploy Multi Site Indexer Cluster and SHC with App framework")
+			Expect(err).To(Succeed(), "Unable to deploy Multisite Indexer Cluster and Search Head Cluster with App framework")
 
-			// Ensure that the CM goes to Ready phase
+			// Ensure that the Cluster Manager goes to Ready phase
 			testenv.ClusterManagerReady(deployment, testenvInstance)
 
-			// Ensure the indexers of all sites go to Ready phase
+			// Ensure the Indexers of all sites go to Ready phase
 			testenv.IndexersReady(deployment, testenvInstance, siteCount)
 
-			// Ensure cluster configured as multisite
+			// Ensure cluster configured as Multisite
 			testenv.IndexerClusterMultisiteStatus(deployment, testenvInstance, siteCount)
 
-			// Ensure SHC go to Ready phase
+			// Ensure Search Head Cluster go to Ready phase
 			testenv.SearchHeadClusterReady(deployment, testenvInstance)
 
 			// Verify RF SF is met
 			testenv.VerifyRFSFMet(deployment, testenvInstance)
 
-			// Verify MC is Ready and stays in ready state
+			// Verify Monitoring Console is Ready and stays in ready state
 			testenv.VerifyMonitoringConsoleReady(deployment, deployment.GetName(), mc, testenvInstance)
 
-			// Verify apps are downloaded by init-container on CM, Deployer
+			//########## INITIAL VERIFICATIONS ##########
+			// Verify V2 apps are downloaded on Cluster Manager and Deployer
 			initContDownloadLocation := "/init-apps/" + appSourceName
 			podNames := []string{fmt.Sprintf(testenv.ClusterManagerPod, deployment.GetName()), fmt.Sprintf(testenv.DeployerPod, deployment.GetName())}
-			testenvInstance.Log.Info("Verify apps are downloaded by init container for apps", "version", appVersion)
+			testenvInstance.Log.Info(fmt.Sprintf("Verify %s apps are downloaded on Cluster Manager and Deployer", appVersion))
 			testenv.VerifyAppsDownloadedOnContainer(deployment, testenvInstance, testenvInstance.GetName(), podNames, appFileList, initContDownloadLocation)
 
-			// Verify apps are downloaded by init-container on Monitoring Console
+			// Verify V2 apps are downloaded on Monitoring Console
 			initContDownloadLocationMCPod := "/init-apps/" + appSourceNameMC
 			mcPodName := fmt.Sprintf(testenv.MonitoringConsolePod, mcName, 0)
-			testenvInstance.Log.Info("Verify apps are downloaded by init container on MC pod for apps", "POD", mcPodName, "version", appVersion)
+			testenvInstance.Log.Info(fmt.Sprintf("Verify %s apps are downloaded on Monitoring Console POD %s", appVersion, mcPodName))
```

Comment:
Log not matching on develop
`testenvInstance.Log.Info(fmt.Sprintf("Verify %s apps are downloaded for Monitoring Console POD %s", appVersion, mcPodName))`